defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 # defaults file for java
 
 java_implementation: openjdk
+java_jre_only: false
+java_headless: false
```

Comment:
These variable names should be more specific.
They both only apply to the OpenJDK installation, and the 'headless' option only applies when installing the JRE only:

java_openjdk_jre_only
java_openjdk_jre_headless